modules/ROOT/partials/install/composer.adoc

```diff
@@ -66,5 +66,5 @@ tinymce.init({
 ----
 
 ____
-If you have decided to use this package manager, move on to the next step link:../work-with-plugins/[working with plugins], where you'll start customizing {productname}. If you'd like to learn about other install options please keep reading.
+If you have decided to use this package manager, move on to the next step xref:general-configuration-guide/work-with-plugins.adoc[working with plugins], where you'll start customizing {productname}. If you'd like to learn about other install options please keep reading.
 ____
```

Comment:
Another admonition that needs converting here.
```suggestion
NOTE: If you have decided to use this package manager, move on to the next step xref:general-configuration-guide/work-with-plugins.adoc[working with plugins], where you'll start customizing {productname}. If you'd like to learn about other install options please keep reading.
```

Reply:
Done